pymc/backends/arviz.py

```diff
@@ -464,7 +468,7 @@ def observed_data_to_xarray(self):
             default_dims=[],
         )
 
-    @requires(["trace", "predictions"])
+    @requires(["trace", "predictions", "prior_predictions"])
```

Comment:
I think this prior predictions attribute is not needed, generating the constant data group here should be enough. IIUC, removing this line completely and leaving everything else untouched should be enough